react/components/admin/pages/Form/Form.tsx

```diff
@@ -220,6 +228,15 @@ const Form: React.FunctionComponent<Props> = ({
         </>
       )}
       <FormFieldSeparator />
+      <Input
+        disabled={!!data.context}
+        label={intl.formatMessage(messages.seoRobots)}
+        onChange={detailChangeHandlerGetter('metaTagRobots')}
+        required
```

Comment:
Should this be required? Considering there's a default value this could be optional. In the typings, we're also using `?` to indicate it's optional.